tests/templates_verification_test.go

```diff
@@ -0,0 +1,82 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2018 Red Hat, Inc.
+ *
+ */
+
+package tests_test
+
+import (
+	"flag"
+	"strings"
+	"net/http"
+	"io/ioutil"
+	"regexp"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/kubevirt/tests"
+)
+
+func search_for_pattern(yaml_segment string, pattern string) bool {
+	r, _ := regexp.Compile(pattern)
+	for _, line := range strings.Split(yaml_segment, "\n") {
+		if r.MatchString(line) {
+			return true
+		}
+	}
+	return false
+}
+
+var _ = Describe("Common templates", func() {
+	flag.Parse()
+
+	BeforeEach(func() {
+		tests.BeforeTestCleanup()
+	})
+
+	Context("Testing generated templates", func() {
+
+		// CNV-1069
+		It("Check if template valid for UI", func() {
+
+			// Getting common_templates
+			// TODO: replace downloading common-templates with getting common-templates from RPM
+			ct_yml_url := "https://github.com/kubevirt/common-templates/releases/download/v0.3.1/common-templates-v0.3.1.yaml"
```

Comment:
Fixed version? What happens when we release a new one?

Reply:
I've implemented determination of the manifest's latest version (using github API)

Reply:
@MarSik ^^